### ISSUE TITLE
Upgrade wizards contain fully qualified class name (with namespaces) instead of short idenfier with underscore

### ISSUE THREAD

**Issue by sypets** [bobosch/ods_osm#104]
The upgrade wizard uses the class as identifier, this was just changed in #103:

```
 public function getIdentifier(): string
    {
 -       return 'odsOsmFileLocationUpdater';
 +       return self::class;
    }
```

```
# add migration wizards
$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update'][\Bobosch\OdsOsm\Updates\FileLocationUpdater::class]
    = \Bobosch\OdsOsm\Updates\FileLocationUpdater::class;
$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update'][\Bobosch\OdsOsm\Updates\MigrateSettings::class]
    = \Bobosch\OdsOsm\Updates\MigrateSettings::class;
```

In any case, it is probably  not the best idea to change the identifier once the extension is out there (since it is usually stored in sys_registry, if the wizard has run already) - but it was already changed recently, in #103. 

The TYPO3 documentation suggests to use an identifier consisting of the extension name, in the example it is: **extName_exampleUpdateWizard**

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ExtensionArchitecture/HowTo/UpdateExtensions/UpdateWizards/Creation.html#registering-wizards

This was changed at some point a while ago. Reason ist, the long identifier with `\` is awkward to call from command line if you want to run a specific upgrade wizard. You also cannot just run it with just the beginning of the name, like you can with composer commands.

---

These are some core upgrade wizards:


`php typo3/sysext/core/bin/typo3  upgrade:list`

 ```
-------------------------------------------------------- ------------------------------------------------------------------------------------------------------- ---------------------------------------------------------------------------- 
  Identifier                                               Title              ....
 -------------------------------------------------------- -------------
  feeditExtension                                          ..... 
                                                                                                                                                                                                                            
  svgFilesSanitization                                   ....
                                                                                                                                                                   
  sysCategorySlugs                                     ....                          
 -------------------------------------------------------- -------------

```

This is the ods_osm upgrade wizard MigrateSettings:

```
php vendor/bin/typo3 upgrade:list
 ---------------------------------------- --------------
  Identifier                               Title           ....
 ---------------------------------------- -------------
  Bobosch\OdsOsm\Updates\MigrateSettings   Migrate settings in EXT:ods_osm plugins   ....
 ---------------------------------------- -------------

```

This is bootstap package (which also still does it with the full class name):

```
php typo3/sysext/core/bin/typo3  upgrade:list
 -------------------------------------------------------- ...
  Identifier                                               Title ...
 --------------------------------------------------------
  BK2K\BootstrapPackage\Updates\CarouselItemLayoutUpdate   [Bootstrap Package] Migrate carouse....
 -------------------------------------------------------- 

```

**Comment by albig** [comment]
Thank you for pointing this.

Indeed, I was looking on the Bootstrap Package but even in the Core, here are upgrade wizards with full class name (`TYPO3\CMS\Felogin\Updates\MigrateFeloginPlugins`). I didn't like this, but thought this is the way to go.

I changed it yesterday, because the wizard identifier in ext_localconf.php was not the one in `FileLocationUpdater::class`. That's why this wizard couldn't run.

So sorry for this. I'would revert this change and fix the ext_localconf.php. But also rename the new `MigrationSettings` wizard.

A patch would be great. Without, I'll fix it this evening.

**Comment by sypets** [comment]
It is more an fyi (for your information). I don't think it is a huge problem but something to consider for the future. 

It is possible to run the wizard using single quotes (but it does pose a difficulty and using quotes, double quotes, backslashes in strings can easily trip people up).

` php vendor/bin/typo3 upgrade:run 'Bobosch\OdsOsm\Updates\MigrateSettings'`

**Comment by albig** [comment]
I reverted back the `odsOsmFileLocationUpdater` wizard, as this was used before. This should not confuse installations where this wizard has been run already. Even if it won't run, if all files have been migrated.

The migration settings wizard has not been published yet. So I renamed it in the same way as the first one. This wizard just helps you to keep the layer configuration in the plugin. Before there was a setting like `[library]_layer` with `library` was one of openlayers, openlayers3, staticmap or leaflet. I changed it to `base_layer` which makes life easier.
